Configs/.config/hypr/hyprlock/boxy.conf

```diff
@@ -0,0 +1,156 @@
+#      ░▒▒▒░░░░░▓▓          ___________
+#    ░░▒▒▒░░░░░▓▓        //___________/
+#   ░░▒▒▒░░░░░▓▓     _   _ _    _ _____
+#   ░░▒▒░░░░░▓▓▓▓▓▓ | | | | |  | |  __/
+#    ░▒▒░░░░▓▓   ▓▓ | |_| | |_/ /| |___
+#     ░▒▒░░▓▓   ▓▓   \__  |____/ |____/
```

Comment:
add a font resolve here for   SF Pro Display

$resolve.font=SF Pro Display|https://font.download/dl/font/sf-pro-display.zip

Reply:
everything else looks good